VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,224 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
      Copyright (C) 2007 The Android Open Source Project
 
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
      You may obtain a copy of the License at
-  
+
           http://www.apache.org/licenses/LICENSE-2.0
-  
+
      Unless required by applicable law or agreed to in writing, software
      distributed under the License is distributed on an "AS IS" BASIS,
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
      See the License for the specific language governing permissions and
      limitations under the License.
+
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" >
+    android:layout_height="match_parent">
 
-    <RelativeLayout
+    <LinearLayout
         android:id="@+id/video_top_bar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentTop="true"
-        android:background="@color/video_details_background_transparent" >
+        android:layout_gravity="top"
+        android:background="@color/video_details_background_transparent"
+        android:gravity="center_vertical"
+        android:orientation="horizontal">
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/video_title"
             style="@style/video_title"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginRight="40dp"
             android:layout_marginEnd="40dp"
             android:layout_marginLeft="5dp"
+            android:layout_marginRight="40dp"
             android:layout_marginStart="5dp"
-            android:text="@string/app_name"
+            android:layout_weight="1"
+            tools:targetApi="17"
+            tools:text="This is a very long video title meant to be truncated from end" />
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/lms_btn"

```

Comment:
yeah about that, I talked with Mike who did the video player changes, he told me iOS doesn't have a share button. I guess @aleffert can explain it better ?